src/OrchardCore.Cms.Web/NLog.config

```diff
@@ -3,7 +3,7 @@
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       autoReload="true"
       internalLogLevel="Warn"
-      internalLogFile="App_Data/logs/internal-nlog.txt">
+      internalLogFile="${var:configDir}/logs/internal-nlog.txt">
```

Comment:
Notice NLog InternalLogger is very simple, and doesn't support advanced layouts like `${var:configDir}`. See also: https://github.com/NLog/NLog/wiki/Internal-Logging (Lists some basic paths)